packages/astro/e2e/fixtures/astro-component/astro.config.mjs

```diff
@@ -1,5 +1,5 @@
-import { defineConfig } from 'astro/config';
 import preact from '@astrojs/preact'
+import { defineConfig } from 'astro/config';
```

Comment:
I think the only slightly odd part of the formatting is that `defineConfig` is now below, but I'm not sure if it's feasible to make a rule to fix this. I'm not too bummed by it but bringing this up in case others might think the same.

Reply:
Makes sense. Here's the algorithm: https://biomejs.dev/analyzer/#how-imports-are-sorted

Reply:
I think the sorting algorithm is fine, but if there's an option to tweak certain orderings, I think that's a great bonus. 
e.g. https://github.com/IanVS/prettier-plugin-sort-imports?tab=readme-ov-file#1-put-specific-dependencies-at-the-top. I used to use this prettier plugin and it has a few options to configure the ordering.